spec/javascripts/Stats/methods_table.spec.js

```diff
@@ -22,15 +21,15 @@ describe('StatsMethodsTable', () => {
     ]
     methodsTable.render()
 
-    let $table = $('table.StatsMethodsTable')
-    let $nameCell = $table.find('.StatsMethodsTable-name').first()
-    let $sinceCell = $table.find('.StatsMethodsTable-since').first()
-    let $untilCell = $table.find('.StatsMethodsTable-until').first()
-    let $totalCell = $table.find('.StatsMethodsTable-total').first()
+    let table = document.querySelector('table.StatsMethodsTable')
+    let nameCell = table.querySelectorAll('.StatsMethodsTable-name')[0]
+    let sinceCell = table.querySelectorAll('.StatsMethodsTable-since')[0]
+    let untilCell = table.querySelectorAll('.StatsMethodsTable-until')[0]
+    let totalCell = table.querySelectorAll('.StatsMethodsTable-total')[0]
```

Comment:
```suggestion
    let nameCell = table.querySelector('.StatsMethodsTable-name')
    let sinceCell = table.querySelector('.StatsMethodsTable-since')
    let untilCell = table.querySelector('.StatsMethodsTable-until')
    let totalCell = table.querySelector('.StatsMethodsTable-total')
```